addons/mail/static/src/new/discuss/discuss.js

```diff
@@ -7,11 +7,16 @@ import { ThreadIcon } from "./thread_icon";
 import { useMessaging } from "../messaging_hook";
 import { Composer } from "../composer/composer";
 import { CallUI } from "../rtc/call_ui";
-import { Component, onWillStart, onMounted, onWillUnmount } from "@odoo/owl";
+import { Component, onWillStart, onMounted, onWillUnmount, useRef } from "@odoo/owl";
+import { CallSettings } from "../rtc/call_settings";
+import { usePopover } from "@web/core/popover/popover_hook";
 
 export class Discuss extends Component {
     setup() {
         this.messaging = useMessaging();
+        this.popover = usePopover();
+        this.closePopover = null;
```

Comment:
note: this pattern is being used quite a lot, maybe we'll need some kind of helper at some point